docs/DeveloperGuide.adoc

```diff
@@ -295,35 +528,291 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` | new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` | event organizer | create a new competition |
+start and organize a new competition
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` | event organizer |find athletes according to their weight class |
```

Comment:
some irregular spacing formatting in this section, like before and after the `|`
not an important issue now, but yeahh

Reply:
i will do it when we finalize the developer guide